ui/src/rdf-vocabularies.ts

```diff
@@ -0,0 +1,44 @@
+import rdf from 'rdf-ext'
+import { Quad } from 'rdf-js'
+import { vocabularies, prefixes, expand as rdfExpand, shrink as rdfShrink } from '@zazuko/rdf-vocabularies'
+
+const relevantPrefixes = ['rdfs', 'schema', 'qb', 'sdmx', 'dcterms', 'dc11', 'skos', 'skosxl', 'xkos', 'xsd', 'wgs']
+
+const PROPERTY = rdf.namedNode(rdfExpand('rdf:Property'))
+const TYPE = rdf.namedNode(rdfExpand('rdf:type'))
+
+export async function loadRDFProperties (): Promise<string[]> {
+  const vocabs = await vocabularies({ only: relevantPrefixes })
+
+  return Object.entries(vocabs).flatMap(([prefix, dataset]) => {
+    const baseIRI = prefixes[prefix]
+    const graph = rdf.namedNode(baseIRI)
+    const properties = dataset.match(null, TYPE, PROPERTY, graph)
+
+    return properties.toArray().map((property: Quad) => shrink(property.subject.value))
+  })
+}
+
+export function expand (uri: string): string {
+  if (uri && !uri.includes(':')) return uri
+
+  if (uri.startsWith('http://')) return uri
```

Comment:
Would at least check for `https` also. If not parse with [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL)

Reply:
"https" didn't cause any issue. But yeah, I could also check for the presence of "://" I guess.